### PR TITLE
[gpgmm] Add gpgmm. Highly experimental.

### PR DESCRIPTION
**Pull request by bbernhar**
Reverts https://github.com/microsoft/vcpkg/pull/25849

Just disables the backend causing the vulkan dependencies (will be fixed in next release).

### CODE REVIEW

File: ports/gpgmm/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  HEAD_REF main
+)
+
+vcpkg_cmake_configure(
+  SOURCE_PATH ${SOURCE_PATH}
+  DISABLE_PARALLEL_CONFIGURE
+  OPTIONS
+  -DGPGMM_STANDALONE=OFF
+  -DGPGMM_ENABLE_TESTS=OFF
+  -DGPGMM_ENABLE_VK=OFF
```
Comment (LilyWangLL) [inline ports/gpgmm/portfile.cmake:15]:
```suggestion
      -DGPGMM_STANDALONE=OFF
      -DGPGMM_ENABLE_TESTS=OFF
      -DGPGMM_ENABLE_VK=OFF

```

File: ports/gpgmm/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,27 @@
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO intel/gpgmm
+  REF v0.0.4
+  SHA512 2ffc3c8299f2d10cb1c0013cd306ba45781a644fa0aa426ef1dfa616e4b53671461a376f65b7068b1ff8a4a2d1a6f9539664174eb5830ea6a760ef5e5d0fc6b0
+  HEAD_REF main
+)
+
+vcpkg_cmake_configure(
+  SOURCE_PATH ${SOURCE_PATH}
```
Comment (LilyWangLL) [inline ports/gpgmm/portfile.cmake:10]:
```suggestion
  SOURCE_PATH "${SOURCE_PATH}"

```

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

File: ports/gpgmm/portfile.cmake (hunk 2 above)
Comment (bbernhar) [reply ports/gpgmm/portfile.cmake]: Done.

File: ports/gpgmm/portfile.cmake (hunk 1 above)
Comment (bbernhar) [reply ports/gpgmm/portfile.cmake]: Done.

**Review by LilyWangLL — APPROVED** [review]

**Comment by bbernhar** [comment]
@LilyWangLL Thank you. Please merge.

**Comment by ras0219-msft** [comment]
Ok this looks good to me. Looking forward to a de-vendored vulkan backend in the future!

**Pull request merged**